backend/src/main/resources/pmd.xml

```diff
@@ -10,6 +10,7 @@
         </properties>
     </rule>
     <exclude-pattern>.*/JiraService.*</exclude-pattern>
+  <exclude-pattern>.*/GitHubService.*</exclude-pattern>
```

Comment:
Why add it

Reply:
because there is  a param ,it said I didn't define, but I define it and used it , this error is just like Guoqing's error before. He exclude JiraService and I did also.